### PR TITLE
chore: updated source paths

### PR DESCRIPTION
**Pull request by Al-Pragliola**
- paths moved from .spec.modules to .spec.distribution.modules
- indent template functions are now dynamic for tolerations/ nodeSelector
- fixed paths to vendor module's folders

### CODE REVIEW

**Review by omissis — APPROVED** [review]

File: source/manifests/aws/kustomization.yaml.tpl [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
-{{- $version := semver .metadata.distributionVersion }}
+{{- $version := semver .spec.distributionVersion }}
 
 resources:
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/cluster-autoscaler/v" $version.Major "." $version.Minor ".x" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/ebs-csi-driver" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/load-balancer-controller" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/node-termination-handler" }}
+  - {{ print "../" .spec.distribution.common.relativeVendorPath "/modules/aws/katalog/cluster-autoscaler/v" $version.Major "." $version.Minor ".x" }}
```
Comment (lnovara) [inline source/manifests/aws/kustomization.yaml.tpl:8]: Why this ended up under a `modules/` directory?

Comment (Al-Pragliola) [reply source/manifests/aws/kustomization.yaml.tpl:8]: I don't know if I can link there the notion page but on the specs for the download phase you can see the folders tree is:
./vendor/modules
./vendor/installers
./vendor/bin

**Review by ralgozino — CHANGES_REQUESTED** [review]
as per previous comment

**Comment by nutellinoit** [comment]
@omissis @Al-Pragliola we need to change the target branch for this PR, this branch `feature/template-eks` is superseded by this one

**Pull request merged**